src/App.js

```diff
@@ -1,4 +1,5 @@
 import React from "react";
+import "bootstrap/dist/css/bootstrap.min.css";
```

Comment:
It seems to be duplicated in `src/index.js`. You might want to take this one out!